samples/matter/common/src/dfu_over_smp.cpp

```diff
@@ -28,28 +30,27 @@ constexpr uint16_t kAdvertisingIntervalMaxMs = 500;
 
 DFUOverSMP DFUOverSMP::sDFUOverSMP;
 
+static mgmt_callback sUploadCallback = {
+	.callback = DFUOverSMP::UploadConfirmHandler,
+	.event_id = MGMT_EVT_OP_IMG_MGMT_DFU_CHUNK,
+};
+
+static mgmt_callback sCommandCallback = {
+	.callback = DFUOverSMP::CommandHandler,
+	.event_id = (MGMT_EVT_OP_CMD_RECV | MGMT_EVT_OP_CMD_DONE),
```

Comment:
It will not work. When the callback is called, individual event or a whole group events is checked.
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/mgmt/mcumgr/mgmt/src/mgmt.c#L109
Here you can use: MGMT_EVT_OP_CMD_ALL

Reply:
Hmm that is actually a bug in the original code then, the idea was that events in a group (but not multiple groups) could be combined with an or for a single event

Reply:
I've opened https://github.com/zephyrproject-rtos/zephyr/issues/53549 will see if I can get it fixed and cherry picked before merge

Reply:
Fix was brought in